css/layout.css

```diff
@@ -30,5 +31,22 @@
 }
 
 .row-timer {
-  padding: 0px;
+  padding-top: 4rem;
+}
+
+.timer {
+  position: absolute;
+  top: -3rem;
+  right: 0rem;
+}
+
+.score-tracker {
+  position: absolute;
+  top: -3rem;
+  left: 0rem;
+}
```

Comment:
These shouldn't need to be absolute positioned, since they are not laying on top of anything 🤔